README.md

```diff
@@ -6,23 +6,35 @@ It's a static site generated by [Jekyll](https://github.com/mojombo/jekyll), and
 
 ## Dependencies
 
-You'll need Jekyll installed to generate and test the site. To get it, most people can install via RubyGems:
+This site uses a Jekyll, a Ruby framework. You'll need Ruby and Bundler installed; see [Jekyll installation instructions](http://jekyllrb.com/docs/installation/) for the details.
 
-    gem install jekyll
+## Building & Viewing
 
-OSX users might have to update RubyGems:
+cd into the directory where you cloned this repository, then install the required gems with `bundle install`. This will automatically put the gems into `./vendor/bundle`.
```

Comment:
Please make 2 sets of instructions. Those for without bundler (most people I'm surrounded by) and for those who wish to build with bundler. 
Arguably lots of people either already have Jekyll installed system-wide on their machine or don't mind installing it system-wide.